test/groovy/NeoDeploymentTest.groovy

```diff
@@ -28,6 +28,10 @@ class NeoDeploymentTest extends PiperTestBase {
         archivePath = "${tmp.newFolder("workspace").toURI().getPath()}archiveName.mtar"
 
         helper.registerAllowedMethod('error', [String], { s -> throw new AbortException(s) })
+        helper.registerAllowedMethod('fileExists', [String], { s ->
```

Comment:
Test would be closer to reality if we hand in the relative path. Inside this method we have to add the leading workspace part which can be derived from the ```setUp``` method. Inside setup we have ```tmp.newFolder('workspace'). This we have to assign to a variable and to prepend inside the fileExists closure. In the test we have to provide the relative path in this case and not the current archivePath.

Otherwise we test here always with the absolute path (... and "fix" it in the mock), but in reality we expect the relative path.

Reply:
Done